packages/google_maps_flutter/google_maps_flutter_ios/pigeons/messages.dart

```diff
@@ -205,6 +206,13 @@ class PlatformMarker {
   final double zIndex;
   final String markerId;
   final String? clusterManagerId;
+  final MarkerCollisionBehavior? collisionBehavior;
+}
+
+enum MarkerCollisionBehavior {
+  required,
```

Comment:
`required` is a keyword: https://dart.dev/language/keywords

Perhaps `requiredDisplay`?